src/backends/zdk-alpha/ZDKAlphaDataSource.ts

```diff
@@ -104,7 +103,7 @@ function getMarkets(markets: TokenMarketResponseItem['markets']) {
   const marketResponse: MarketModule[] = [];
   markets.forEach((market) => {
     const getStandardMarketData = (
-      market: TokenMarketResponseItem['markets'][0],
+      market: MarketInfoFragment & MarketDetailsFragment,
```

Comment:
Is there a reason why this changed from TokenResponseItem['markets'][0]? This works, but another option is to create a new type.

Reply:
Fixed